tests/queries/0_stateless/03203_json_empty_as_default.sql

```diff
@@ -0,0 +1,53 @@
+set input_format_json_empty_as_default = 1;
+set allow_experimental_variant_type = 1;
+
+# Simple types
```

Comment:
Could you, please, add string "-- { echoOn }" here. This helps with separating tests from each other